### PR TITLE
Added workflow to package and ship scrape2md to npm

### PR DESCRIPTION
**Pull request by Rachit1313**

### CODE REVIEW

**Comment by Rachit1313** [comment]
@tarasglek , Could you please setup NPM_TOKEN in the repo's secret, create a release and also review this PR?

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '18'
```
Comment (humphd) [inline .github/workflows/npm-publish.yml]: Node 20 please

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install pnpm
```
Comment (humphd) [inline .github/workflows/npm-publish.yml]: use the pnpm action:

```yaml
            - uses: pnpm/action-setup@v2
              with:
                  version: 8
```

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "scrape2md",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": " A package to scrape web content into readable markdown for llms and human readers",
+  "main": "./src/index.ts",
```
Comment (humphd) [inline package.json]: You need to build this to JS.  I would recommend looking at what Yumei did with the other Typescript2Openai repo.

**Review by humphd — CHANGES_REQUESTED** [review]

**Comment by tarasglek** [comment]
image

**Review by tarasglek — APPROVED** [review]
lets try this

**Pull request merged**